docs/GETTING_STARTED.rst

```diff
@@ -1,9 +1,8 @@
 Getting Started
 ---------------
 
+- `Overview of TUF <OVERVIEW.rst>`_
 - `Installation <INSTALLATION.rst>`_
-- `Contributors <CONTRIBUTORS.rst>`_
-- `Quickstart <QUICKSTART.md>`_
-- `CLI <CLI.md>`_
-- `CLI Usage Examples <CLI_EXAMPLES.md>`_
-- `Tutorial <TUTORIAL.md>`_
+- Beginner Tutorials (using the basic command-line interface): `Quickstart <QUICKSTART.md>`_; `CLI Tutorial <CLI.md>`_; `CLI Further Examples <CLI_EXAMPLES.md>`_
+- `Advanced Tutorial <TUTORIAL.md>`_
```

Comment:
If you call it advanced tutorial here, which I think is a good idea, you maybe also want to call it that in the document heading?

Reply:
Fixed there and elsewhere, thanks.